multi-transfer-esdt/src/lib.rs

```diff
@@ -348,19 +411,38 @@ pub trait MultiTransferEsdt:
             }
             ManagedAsyncCallResult::Err(_) => {
                 self.unwrap_tokens(&tx.token_id, refund_payment.clone());
-                let refund_tx = self.convert_to_refund_tx(tx.clone());
+                let refund_tx = self.convert_to_tx(tx.clone());
                 self.add_to_batch(refund_tx);
 
                 self.transfer_failed_frozen_destination_account_event(batch_id, tx.tx_nonce);
             }
         }
     }
 
+    //views
+
+    #[view(getTransactionForBlacklistTokens)]
+    fn get_transaction_for_blacklist_tokens(
+        &self,
+    ) -> MultiValueEncoded<MultiValue2<u64, EthTransaction<Self::Api>>> {
+        let mut transactions = MultiValueEncoded::new();
+        for (tx_id, tx) in self.transactions_for_blacklist_tokens().iter() {
```

Comment:
Can't this storage get too big so that querying it from a different SC will get to an outOfGas situation? It doesn't receive a parameter or any kind of filtering.

Reply:
This is only a view function. No impact on-chain.